account/user.go

```diff
@@ -115,6 +118,18 @@ func (m *GormUserRepository) Delete(ctx context.Context, id uuid.UUID) error {
 	return nil
 }
 
+// List return all users
+func (m *GormUserRepository) List(ctx context.Context) ([]*User, error) {
+	defer goa.MeasureSince([]string{"goa", "db", "user", "list"}, time.Now())
+	var rows []*User
+
+	err := m.db.Model(&User{}).Order("email").Find(&rows).Error
```

Comment:
just curious: what happens when the DB contains thousands of identities: do we really want to return all the data at once ? Shouldn't we require pagination to limit the amount of returned data ?